test/sessionExpress.test.js

```diff
@@ -174,59 +174,6 @@ describe(`sessionExpress: ${printPath("[test/sessionExpress.test.js]")}`, functi
         assert(res2.status === 404);
     });
 
-    it("test that the sign out API clears cookies without session", async function () {
```

Comment:
this test is still useful to have.

Reply:
It'd fail. Should I re-add it with testing that we do not clear cookies?